docker/templates/alpine/base.Dockerfile.j2

```diff
@@ -0,0 +1,52 @@
+{{ metadata.header }}
```

Comment:
Currently I was thinking to structured our generated image directory as below:
``` bash
generated
  - model-server
    - python_version
      - {distro}{distro_version}
         - Dockerfile (this is the base image)
         - cudnn
         - devel
         - runtime 
```
This is just a personal preference, where I would like to define all build_args into `ENV`and have unique Dockerfile for each releases.

Also it would be better for debugging.

Reply:
probably parsed python_version at build-time to reduce boilerplate

Reply:
There are a lot of similarity between centos and ubuntu releases (difference are pretty much os version). This is due to `docker.api.build` requires a file object for Dockerfile to be found in given directory to build the image.

I was thinking to use 
```docker
ARG OS_IMAGE

FROM $OS_IMAGE
``` 
and generate from one Dockerfile (eg: for both `ubuntu20.04` and `ubuntu18.04`). However, the current implementation helps easier debugging and exploring image